gammapy/modeling/models/time.py

```diff
@@ -246,3 +316,71 @@ def mean_norm_in_time_interval(self, time_min, time_max):
         dt = time_max - time_min
         integral = self._interpolator.integral(time_min, time_max)
         return integral / dt
+
+    #    @property
+    def ontime(self, t_min, t_max):
+        """On time (`~astropy.units.Quantity`)"""
+
+        t_min = Time(t_min)
+        t_max = Time(t_max)
+
+        ontime = t_max - t_min
+        return u.Quantity(ontime.sec, "s")
+
+    def _get_time_meta(self, t_min, t_max):
```

Comment:
As far as I can tell this method is unused. Please remove.

Reply:
Done